.github/workflows/contrib-openai.yml

```diff
@@ -53,7 +58,7 @@ jobs:
           AZURE_OPENAI_API_BASE: ${{ secrets.AZURE_OPENAI_API_BASE }}
           OAI_CONFIG_LIST: ${{ secrets.OAI_CONFIG_LIST }}
         run: |
-          coverage run -a -m pytest test/agentchat/contrib/test_retrievechat.py::test_retrievechat test/agentchat/contrib/test_qdrant_retrievechat.py::test_retrievechat
+          coverage run -a -m pytest test/agentchat/contrib/test_retrievechat.py::test_retrievechat test/agentchat/contrib/test_qdrant_retrievechat.py::test_retrievechat test/agentchat/contrib/test_pgvector_retrievechat.py::test_retrievechat
```

Comment:
To run the added test, extra dependency is needed.
Also, since the setup.py is changed, the package installation section should be updated.

Reply:
I have the dependency added on line 44 of contrib-openai.yml. Did I misunderstand?

Which package installation section? I don't see any required changes in python-pacakge.yml workflow. I also did update the Optional-Depenncies in the installation docs.

Reply:
There is a "retrievechat-pgvector" option added in setup.py. Line 44 installs postgresql.
I thought that Line 49 is a package installation section that should be updated. "retrievechat-qdrant" and "retrievechat-pgvector" options are supposed to be installed there, if I understand it correctly. Please let me know if I misunderstood.

Reply:
I see now what you mean. Thank you for calling that out. I have fixed this.